.jenkins

```diff
@@ -44,7 +44,7 @@ pipeline {
                                 -DCMAKE_BUILD_TYPE=Release \
                                 -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
                                 -DCMAKE_CXX_COMPILER=clang++ \
-                                -DCMAKE_CXX_FLAGS="-Werror -Wno-unknown-cuda-version -Wno-gnu-zero-variadic-macro-arguments" \
+                                -DCMAKE_CXX_FLAGS="-Werror -Wno-gnu-zero-variadic-macro-arguments -Wno-linker-warnings" \
                                 -DKokkos_ARCH_VOLTA70=ON \
                                 -DKokkos_ENABLE_COMPILER_WARNINGS=ON \
                                 -DKokkos_ENABLE_DEPRECATED_CODE_3=ON \
```

Comment:
There seems to be some problem with `impl/Kokkos_Timer.hpp` being included instead of `Kokkos_Timer.hpp` for `#include <Kokkos_Timer.hpp>`. I can't reproduce, though, and would want to move forward here independent of this weirdness.

Reply:
See https://github.com/intel/llvm/issues/4226.